src/main/java/com/twilio/oai/TwilioGoGenerator.java

```diff
@@ -21,6 +21,13 @@ public void processOpts() {
     public void postProcessParameter(final CodegenParameter parameter) {
         // Make sure required non-path params get into the options block.
         parameter.required = parameter.isPathParam;
+        parameter.vendorExtensions.put("x-is-account-sid", this.toLowerCase(parameter.baseName).equals("accountsid"));
+    }
+
+    private String toLowerCase(String paramName) {
+        // pre-process name
+        paramName = paramName.replaceAll("[^a-zA-Z]", "");
```

Comment:
Is this needed, or is it always just `AccountSid`?

Reply:
hmm right now it's always `AccountSid`. I'll get rid of this.